lib/create-space-api.js

```diff
@@ -513,6 +514,81 @@ export default function createSpaceApi ({
   }
 
   /**
+   * Creates a Asset based on files. After creation, call asset.processForLocale or asset.processForAllLocales to start asset processing.
+   * @memberof ContentfulSpaceAPI
+   * @see {Asset.Asset}
+   * @param {object} data - Object representation of the Asset to be created. Note that the field object should have an uploadFrom property on asset creation, which will be removed and replaced with an url property when processing is finished.
+   * @param {object} data.fields.file.[LOCALE].file - Can be a string, an ArrayBuffer or a Stream.
+   * @return {Promise<Asset.Asset>} Promise for the newly created Asset
+   * @example
+   * space.createAssetFromFiles({
+   *   fields: {
+   *     file: {
+   *       'en-US': {
+   *          contentType: 'image/jpeg',
+   *          fileName: 'filename_english.jpg',
+   *          file: readFileSync('path/to/filename_english.jpg')
```

Comment:
Axios accepts multiple formats, but yes, the example should use a less memory consuming version :) 👍